data-prepper-plugins/otel-trace-raw-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/oteltrace/OTelTraceRawProcessor.java

```diff
@@ -36,7 +36,7 @@
 import java.util.stream.Collectors;
 
 
-@DataPrepperPlugin(name = "otel_trace_raw", pluginType = Processor.class, pluginConfigurationType = OtelTraceRawProcessorConfig.class)
+@DataPrepperPlugin(name = "otel_traces", deprecatedName = "otel_trace_raw", pluginType = Processor.class, pluginConfigurationType = OtelTraceRawProcessorConfig.class)
```

Comment:
I'm not sure that this is the new name is the right name to have here (I also think the old name was probably not ideal). We have three different processors for traces, so I think we might want to be clearer on what we use each for.

I think I'd recommend that we drop this from this PR and start a new issue to resolve the names for these plugins.

Reply:
I agree, I wanted to discuss that as well in the PR feedback. I'm not convinced either that it's correct name. I prefer having a distinction between source and processor name.

I will create an issue for it

Reply:
+1 would like to differentiate between source and processor name, but not sure what that would be either as this description aligns more with the `otel_trace_group` name 

```
This is a processor that receives collection of [Span](../../data-prepper-api/src/main/java/com/amazon/dataprepper/model/trace/Span.java) records sent from [otel-trace-source](../dataPrepper-plugins/otel-trace-source), does stateful processing on extracting and filling-in trace group related fields.

```